tests/nested/core/core22-basic/task.yaml

```diff
@@ -0,0 +1,28 @@
+summary: Run a smoke test on UC22 with encryption enabled
```

Comment:
can we do more of the checks that encryption is turned on / usable here too? like this code at least:
```bash
    echo "Ensure 'snap debug show-keys' works as root"
    tests.nested exec "sudo snap recovery --show-keys" | MATCH 'recovery:\s+[0-9]{5}-[0-9]{5}-[0-9]{5}-[0-9]{5}-[0-9]{5}-[0-9]{5}-[0-9]{5}-[0-9]{5}'
    tests.nested exec "sudo snap recovery --show-keys" | MATCH 'reinstall:\s+[0-9]{5}-[0-9]{5}-[0-9]{5}-[0-9]{5}-[0-9]{5}-[0-9]{5}-[0-9]{5}-[0-9]{5}'
    echo "But not as user (normal file permissions prevent this)"
    if tests.nested exec "snap recovery --show-key"; then
        echo "snap recovery --show-key should not work as a user"
        exit 1
    fi
```

Reply:
sure, done